app/src/main/res/layout/fragment_detail.xml

```diff
@@ -127,6 +127,15 @@
                     android:textSize="18sp"
                     tools:text="Article Author Content" />
 
+                <ImageView
+                    android:id="@+id/iv_img_url"
+                    android:layout_width="wrap_content"
```

Comment:
Make sure to set the width here to `match_parent`, otherwise the view will attempt to take up the full available height of the screen too.

You should also change the scaleType to `fitXY` in this case as well.

Reply:
Will do. Thanks!